packages/common/src/types/savedCharts.ts

```diff
@@ -339,9 +340,13 @@ export const getChartType = (
                     case CartesianSeriesType.AREA:
                         return ChartKind.AREA;
```

Comment:
as @ZeRego mentioned area chart is line chart + `value.eChartsConfig.series?.[0]?.areaStyle` so that means this switch case never gets matched because `value.eChartsConfig.series?.[0]?.type` will never be area?

Reply:
> means this switch case never gets matched because value.eChartsConfig.series?.[0]?.type will never be area?
Correct, will be line

Reply:
`CartesianSeriesType.AREA` is referenced in different places with different uses and is not a simple change. so I think we should do it out of the scope of this PR